pages/docs/core-concepts/create-and-run-effects.mdx

```diff
@@ -44,9 +46,9 @@ In the sections below, we walk through some of the more common use cases.
 To create an `Effect` from a value, you can use the `succeed` constructor:
 
 ```ts twoslash
-import * as Effect from "@effect/io/Effect"
+import * as Effect from "@effect/io/Effect";
```

Comment:
I would prefer we leave the semicolons out of all code samples - let's go ahead and remove them everywhere. The less syntax noise the better.

Reply:
@IMax153 The only case when semicolons are visible is when someone explicitly mentions them. Or when the lack of them breaks prettier.

Reply:
Is this not considered "explicitly mentioning them"?